frontend/src/common/HelpDialog.tsx

```diff
@@ -1,37 +1,29 @@
 import HelpOutlineIcon from '@mui/icons-material/HelpOutline'
-import { Button, Dialog, DialogContent, DialogTitle, Slide, Typography } from '@mui/material'
-import { TransitionProps } from '@mui/material/transitions'
-import React, { ReactNode } from 'react'
+import { Button, Dialog, DialogActions, DialogContent, DialogTitle, IconButton } from '@mui/material'
+import { ReactNode, useState } from 'react'
 
-const Transition = React.forwardRef(function Transition(
-  props: TransitionProps & {
-    children: React.ReactElement<any, any>
-  },
-  ref: React.Ref<unknown>,
-) {
-  return <Slide direction='up' ref={ref} {...props} />
-})
-
-type Props = {
-  infoLabel: string
+type HelpDialogProps = {
   dialogTitle: string
   content: ReactNode
-  icon?: ReactNode
 }
 
-function HelpDialog({ infoLabel, dialogTitle, content, icon }: Props) {
-  const [open, setOpen] = React.useState(false)
+export default function HelpDialog({ dialogTitle, content }: HelpDialogProps) {
+  const [open, setOpen] = useState(false)
 
   return (
     <>
-      <Button size='small' onClick={() => setOpen(true)} endIcon={icon ? icon : <HelpOutlineIcon />}>
-        <Typography variant='caption'>{infoLabel}</Typography>
-      </Button>
-      <Dialog open={open} onClose={() => setOpen(false)} maxWidth='md' TransitionComponent={Transition}>
+      <IconButton size='small' onClick={() => setOpen(true)}>
```

Comment:
I think this got lost in one of my other comments, but I think this would benefit from a tooltip. We could simply repurpose the `dialogTitle`, rename it to `title` and use that title for the tooltip and dialog.
```
  return (
    <>
      <Tooltip title={title}>
        <IconButton size='small' onClick={() => setOpen(true)}>
          <HelpOutlineIcon />
        </IconButton>
      </Tooltip>
      <Dialog open={open} onClose={() => setOpen(false)} maxWidth='md' disableEscapeKeyDown>
        <DialogTitle>{title}</DialogTitle>
        <DialogContent>{content}</DialogContent>
        <DialogActions>
          <Button color='secondary' variant='outlined' onClick={() => setOpen(false)}>
            Close
          </Button>
        </DialogActions>
      </Dialog>
    </>
  )
```